File: src/main/java/no/nav/pensjon/regler/domain/beregning/FasteUtgifterTillegg.java
package no.nav.pensjon.regler.domain.beregning;

import no.nav.pensjon.regler.domain.kode.YtelsekomponentTypeCti;

/**
 * Tillegget for faste utgifter. Brukes ved institusjonsopphold.
 */
public class FasteUtgifterTillegg extends Ytelseskomponent {

    private static final long serialVersionUID = -2607047532936249148L;

    public FasteUtgifterTillegg() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("FAST_UTGIFT_T");
    }

    public FasteUtgifterTillegg(int fasteUtgifter) {
        super();
        netto = fasteUtgifter;
    }

    public FasteUtgifterTillegg(FasteUtgifterTillegg fasteUtgifterTillegg) {
        super(fasteUtgifterTillegg);
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning/FasteUtgifterTillegg.kt
package no.nav.pensjon.regler.domain.beregning

import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

/**
 * Tillegget for faste utgifter. Brukes ved institusjonsopphold.
 */
class FasteUtgifterTillegg : Ytelseskomponent() {
    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.FAST_UTGIFT_T
}